website/src/documentation/runners/jet.md

```diff
@@ -59,24 +59,58 @@ Issue following command in the Beam examples project to start new Jet cluster an
 ```
 
 ### Running WordCount on a Remote Jet Cluster ##
-Download latest stable Hazelcast Jet code from [Hazelcast Website](https://jet.hazelcast.org/download/) and 
-start Jet cluster. 
-The simplest way is to start Jet cluster member using the `jet-start` script that comes with Jet distribution.
-The members use the [auto discovery feature](https://docs.hazelcast.org/docs/3.12/manual/html-single/index.html#setting-up-clusters) 
+Download latest stable Hazelcast Jet version, that Beam depends on, from 
+[Hazelcast Jet Website](https://jet-start.sh/download). At the time of writing the 
+[latest Beam release]({{site.baseurl}}/get-started/downloads/#releases) is 2.19.0, which still depends on Hazelcast
+Jet 3.0, but following versions should migrate to Jet 4.0.
```

Comment:
Should we include a version table here similar to the Flink Runner page? I would not hardcode anything regarding the Beam release at the time of the writing.

Reply:
Did so, much better indeed. Thanks